docs/readmes/orc8r/hold_terraform_state_on_s3.md

```diff
@@ -0,0 +1,100 @@
+---
+id: hold_terraform_state_on_s3
+title: Hold Terraform state outside my local machine
+hide_title: true
+---
+# Holding the terraform state outside of my local machine
```

Comment:
moving this KB to github so it can be visible to open source community. FYI  @sudhikan since you helped preparing this doc.